numpy/lib/twodim_base.py

```diff
@@ -136,11 +205,11 @@ def flipud(m):
     return m[::-1, ...]
 
 
-def rot90(m, k=1):
+def rot90(m, k=1, axes=(0,1)):
     """
     Rotate an array by 90 degrees in the counter-clockwise direction.
```

Comment:
This needs a better description. CCW does not really apply to an N-D array. Perhaps the order of the axes should be mentioned here. I would expect that `rot90(m, axes=(0, 1))` would be the opposite of `rot90(m, axes=(1, 0))` but the same as `rot90(m, k=-1, axes=(1, 0))`. I am pretty sure that is what your function does, but I think you should make the docs and the tests explicit for both reversed `axes` and negative `k`.